app/views/va_facilities/_facility_categories_combo_box.html.erb

```diff
@@ -1,4 +1,4 @@
-<label for="<%= combo_box_id %>" class="usa-label margin-top-0 margin-bottom-05">
+<label for="<%= combo_box_id %>" class="usa-label margin-top-0 margin-bottom-0">
```

Comment:
That seems to throw off the comb box slightly from being centered with the key word search box for both the practices created and adopted practices.. since they are using the same partial.

Reply:
Done